<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MP3音频播放器--测试版</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/images/0.ico</normaloff>:/new/prefix1/images/0.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>711</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>歌曲列表</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>711</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_open">
       <property name="text">
        <string>打开</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/new/prefix1/images/1.GIF</normaloff>:/new/prefix1/images/1.GIF</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_PLayer">
       <property name="text">
        <string>播放</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/new/prefix1/images/4.bmp</normaloff>:/new/prefix1/images/4.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Pause">
       <property name="text">
        <string>暂停</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/new/prefix1/images/5.bmp</normaloff>:/new/prefix1/images/5.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Stop">
       <property name="text">
        <string>停止</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/new/prefix1/images/6.bmp</normaloff>:/new/prefix1/images/6.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Prev">
       <property name="text">
        <string>上一曲</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/new/prefix1/images/7.bmp</normaloff>:/new/prefix1/images/7.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_next">
       <property name="text">
        <string>下一曲</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/new/prefix1/images/8.bmp</normaloff>:/new/prefix1/images/8.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Volumn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/new/prefix1/images/volumn.bmp</normaloff>:/new/prefix1/images/volumn.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_Volumn">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>711</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_Name">
       <property name="text">
        <string>歌曲名称</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_Speed">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_Time">
       <property name="text">
        <string>播放进度</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>listWidget</zorder>
   <zorder>pushButton_open</zorder>
   <zorder>pushButton_PLayer</zorder>
   <zorder>pushButton_Pause</zorder>
   <zorder>pushButton_Stop</zorder>
   <zorder>pushButton_Prev</zorder>
   <zorder>pushButton_next</zorder>
   <zorder>pushButton_Volumn</zorder>
   <zorder>horizontalSlider_Volumn</zorder>
   <zorder>label_Name</zorder>
   <zorder>horizontalSlider_Speed</zorder>
   <zorder>label_Time</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
